Common/src/main/java/com/tabnineCommon/chat/commandHandlers/context/DiagnosticsContext.kt

```diff
@@ -12,8 +12,10 @@ import com.tabnineCommon.chat.commandHandlers.utils.AsyncAction
 import java.awt.Point
 import java.util.concurrent.CompletableFuture
 
+data class DiagnosticItem(val errorMessage: String, val lineNumber: Int, val lineCode: String)
+
 data class DiagnosticsContext(
-    private val diagnosticsText: String? = null,
+    private val diagnostics: List<DiagnosticItem>? = null,
```

Comment:
how are you handling backwards compatibility in the app and FE?

Reply:
this way
https://github.com/codota/tabnine-chat-frontend/blob/main/src/services/userContextPrompts.ts#L101